CONTRIBUTORS.yaml

```diff
@@ -0,0 +1,252 @@
+# List of contributors
+#
+# reference maintainers/contributors by their id in tutorial metadata files
+#
+# Institutional sponsors
+#  * Collaborative Research Centre (CRC) 992 - Medical Epigenetics
+#  * de.NBI - German Network for Bioinformatics Infrastructure
+#  * NHGRI - National Human Genome Research Institute at the National Institutes of Health (USA)
+#  * NSF - National Science Foundation (USA)
+#
+---
+abretaud:
+    name: Anthony Bretaudeau
+    github_username: abretaud
```

Comment:
Is there not a bit of redundancy the `github_username`? Are they different from the keys of the list?

Reply:
for the vast majority yes, but there are a couple of people in the list without a github username, guess they contributed to the materials indirectly (Andreas Richter, Torsten Houwaart)

Reply:
guess we could make a "github: no" key for these people and for all others assume it is same as the key?

Reply:
@TorHou is Torsten

Reply:
For Torsten, the github username is TorHou
Your idea sounds good :)

Reply:
ah thanks!